src/main/java/com/elytradev/concrete/block/ConcreteBlock.java

```diff
@@ -0,0 +1,206 @@
+package com.elytradev.concrete.block;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.item.Item;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import net.minecraftforge.fml.common.FMLContainer;
+import net.minecraftforge.fml.common.FMLLog;
+import net.minecraftforge.fml.common.InjectedModContainer;
+import net.minecraftforge.fml.common.Loader;
+import net.minecraftforge.fml.common.ModContainer;
+
+import java.util.Optional;
+import java.util.Random;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+/**
+ * An extension of {@link Block} used by the Concrete block builder system, to allow for
+ * the create of blocks through said system.
+ *
+ * This is accomplished by overriding necessary methods from {@link Block}, and replacing
+ * them with the various behaviours that are defined by the user or the ones provided by
+ * Concrete.
+ */
+public class ConcreteBlock extends Block {
+
+    /**
+     * A preset for a block builder with ore properties, such as the fortune drop behaviour.
+     *
+     * @see Builder#preset(Consumer)
+     */
+    public static Consumer<Builder> ORE_PRESET = builder -> builder.drop(ItemDropBehaviour.Fortune.DEFAULT);
```

Comment:
These should probably go into a `Presets` class, instead of just being thrown in the block one.